common/reducers/Prefs.js

```diff
@@ -22,9 +22,6 @@ const INITIAL_STATE = {
 module.exports = function Prefs(prevState = INITIAL_STATE, action) {
   const state = Object.assign({}, prevState);
   switch (action.type) {
-    case am.type("PREFS_REQUEST"):
-      state.isLoading = true;
```

Comment:
Looks like `isLoading` is always `false` now and is effectively unused... And a quick glance seems to show `init` is also unused?